### ISSUE TITLE
Support for Express, Node and Angular is unreliable

### ISSUE THREAD

**Issue by shamilovtim** [gdelmas/IntelliJDashPlugin#64]
1) Activating shortcut on a method which belongs to Express, Node or Angular.
2) WebStorm passes the Library to Dash as "Javascript" despite docsets existing for these libraries.

Plugin is a bit too simplistic right now to be fully functional. We must pass the appropriate libraries to their own docsets rather than looking up these lib functions in the javascript docset.

**Comment by gdelmas** [comment]
hi timothy,

the plugin passes a list of filter keywords to dash when invoking a query. this list is compiled by the dash developer and can be found here: https://kapeli.com/dash_plugins. additional keywords have been added to the plugin by contributors.

this means that when triggering on javascript code dash will also lookup docsets like `angularjs`, `express` and `nodejs`.

apart from that docsets for javascript frameworks or libraries should also be query-able with the javascript keyword. dash checks several fields in the docset metadata. one of these is `DashDocSetPluginKeyword`.

in conclusion: an invocation on javascript code searches express, node, angular, javascript and many other docsets.
